### PR TITLE
feat: add POST `/ssh-keypair` to save ssh keypair

### PR DESCRIPTION
**Pull request by yomybaby**
ref https://github.com/lablup/giftbox/issues/58

### CODE REVIEW

File: src/ai/backend/manager/api/auth.py [hunk 1]
```diff
@@ -992,6 +992,34 @@ async def refresh_ssh_keypair(request: web.Request) -> web.Response:
...
+@auth_required
+@check_api_params(
+    t.Dict(
+        {
+            t.Key("pubkey"): t.String,
+            t.Key("privkey"): t.String,
+        }
+    )
+)
+async def save_ssh_keypair(request: web.Request, params: Any) -> web.Response:
```
Comment (kyujin-cho) [inline src/ai/backend/manager/api/auth.py]: How about renaming this method to `upload_ssh_keypair` and `refresh_ssh_keypair` to `generate_ssh_keypair` to make it clear?

**Review by kyujin-cho — CHANGES_REQUESTED** [review]

File: src/ai/backend/manager/api/auth.py (hunk 1 above)
Comment (yomybaby) [reply src/ai/backend/manager/api/auth.py]: Great suggestion. I'll change to `upload_ssh_keypair` and `generate_ssh_keypair`

Comment (yomybaby) [reply src/ai/backend/manager/api/auth.py]: I just pushed renamings.

**Review by kyujin-cho — APPROVED** [review]

File: changes/1032.feature.md [hunk 2]
```diff
@@ -0,0 +1 @@
+add POST `/ssh/keypair` api to save ssh keypair manually
```
Comment (kyujin-cho) [inline changes/1032.feature.md]:
```suggestion
add POST `/ssh/keypair` API to let user use their own SSH keypair instead of randomly generated one
```

**Review by kyujin-cho — CHANGES_REQUESTED** [review]

**Pull request merged**